charts/dispatch/charts/function-manager/templates/config-map.yaml

```diff
@@ -21,7 +22,7 @@ data:
           "funcNamespace": "{{ .Values.faas.openfaas.namespace }}"
         },
         "riff": {
-          "gateway": "{{ .Values.faas.riff.gateway }}",
+          "kafkaBrokers": ["{{ index .Values.faas.riff.kafkaBrokers 0 }}"],
```

Comment:
helm uses [sprig](https://godoc.org/github.com/Masterminds/sprig), so this should work:
```
"kafkaBrokers": ["{{ join "," .Values.faas.riff.kafkaBrokers }}"],
```

Reply:
Thanks! Fixed